lib/node_modules/@stdlib/stats/base/dists/weibull/mgf/benchmark/benchmark.js

```diff
@@ -32,17 +33,25 @@ var mgf = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var lambda;
+	var len;
 	var k;
 	var t;
 	var y;
 	var i;
 
+	len = 100;
+	t = new Float64Array( len );
+	lambda = new Float64Array( len );
+	k = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		t[ i ] = uniform( EPS, 2.0 );
```

Comment:
```suggestion
		t[ i ] = uniform( EPS, 1.0 );
```

Make sure the minimum and maximum values in the `uniform` expressions are no different than the original ones.